source/auth/auth.rst

```diff
@@ -834,10 +834,9 @@ Client Second:
        "conversationId" : 1,
        "payload" : new BinData(0, "LQEAAAJhAAkBAABBV1M0LUhNQUMtU0hBMjU2IENyZWRlbnRpYWw9QUtJQUlDR1ZMS09LWlZZM1gzREEvMjAxOTExMTIvdXMtZWFzdC0xL3N0cy9hd3M0X3JlcXVlc3QsIFNpZ25lZEhlYWRlcnM9Y29udGVudC1sZW5ndGg7Y29udGVudC10eXBlO2hvc3Q7eC1hbXotZGF0ZTt4LW1vbmdvZGItZ3MyLWNiLWZsYWc7eC1tb25nb2RiLXNlcnZlci1ub25jZSwgU2lnbmF0dXJlPThhMTI0NGZjODYyZTI5YjZiZjc0OTFmMmYwNDE5NDY2ZGNjOTFmZWU1MTJhYTViM2ZmZjQ1NDY3NDEwMjJiMmUAAmQAEQAAADIwMTkxMTEyVDIxMDEyMloAAA==")
    }
-|
 
 In response to the Server First message, drivers MUST send an ``authorization header``. Drivers MUST follow the
-`Signature Version 4 Signing Process <https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html>`_ to
+`Signature Version 4 Signing Process <https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html>`__ to
```

Comment:
Why do these links need a double underscore? AFAIK links only have one underscore: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks

Reply:
This is a subtle thing that I didn't know about until I saw warnings about it from Sphinx. A single underscore generates a named hyperlink target for the whole page, where the name of the target is the link text. There are two links in the page with the same link text but different URLs, and rST needs to pick one. Using two underscores generates an anonymous link, so no link target name is globally defined, and they are separate links.